lib/DirectAdmin_API.php

```diff
@@ -69,7 +87,7 @@ public function getDomainsMxRecords($domain){
 				$records[] = array(
 					'original'	=> $fixedRecord,
 					'full'		=> substr($fixedRecord, -1) == '.' ? substr($fixedRecord, 0, -1) : $fixedRecord . '.' . $domain,
-					'isRemote'	=> substr($fixedRecord, -1) == '.' && strpos($fixedRecord, $domain) === false ? true : false
+					'isRemote'	=> $isRemote,
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->